### PR TITLE
Update Caddy image's tag

### LINKED ISSUE #1398: Update Caddy image to 2.2 for production build
Hi @sobolevn ,

I would like to propose you to use Caddy `2.2` in this [line of code](https://github.com/wemake-services/wemake-django-template/blob/421fd2325cd293bde598654fe6f2608dbef41927/%7B%7Bcookiecutter.project_name%7D%7D/docker/docker-compose.prod.yml#L12). What do you think about it?

### PR DESCRIPTION
**Pull request by DmytroLitvinov**
From [DockerHub of Caddy image](https://hub.docker.com/_/caddy) there is a shared tag **2** which we can use to use get latest stable release.

Closes #1398

### CODE REVIEW

File: {{cookiecutter.project_name}}/docker/docker-compose.prod.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 version: "3.6"
 services:
   caddy:
-    image: "caddy:2.0.0"
+    image: "caddy:2"
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/docker/docker-compose.prod.yml:12]: Let's use `image: "caddy:2.2"`

**Review by sobolevn — CHANGES_REQUESTED** [review]
Thanks! 

Are there any new features we can add to our `Caddyfile` / config?

File: {{cookiecutter.project_name}}/docker/docker-compose.prod.yml (hunk 1 above)
Comment (DmytroLitvinov) [inline {{cookiecutter.project_name}}/docker/docker-compose.prod.yml:12]:
```suggestion
    image: "caddy:2.2"
```

Comment (DmytroLitvinov) [reply {{cookiecutter.project_name}}/docker/docker-compose.prod.yml:12]: @sobolevn , there is no caddy@2.2 as a shared tag. Only next available tags: `2.2.1`, `2`, `latest`. More details at [DockerHub](https://hub.docker.com/_/caddy)

**Comment by DmytroLitvinov** [comment]
> Are there any new features we can add to our `Caddyfile` / config?

I don't have enough experience with Caddy so I am not sure that I can suggest something here. From [changelog](https://github.com/caddyserver/caddy/releases) I see there is nothing special that we should add or we can improve. 
Would be nice if you can also fact-check it and confirm it.

File: {{cookiecutter.project_name}}/docker/docker-compose.prod.yml (hunk 1 above)
Comment (sobolevn) [reply {{cookiecutter.project_name}}/docker/docker-compose.prod.yml:12]: Packaging is hard 😒

Comment (sobolevn) [reply {{cookiecutter.project_name}}/docker/docker-compose.prod.yml:12]: `2.2.1` then!

Comment (DmytroLitvinov) [inline {{cookiecutter.project_name}}/docker/docker-compose.prod.yml:12]:
```suggestion
    image: "caddy:2.2.1"
```

**Review by sobolevn — APPROVED** [review]

**Comment by sobolevn** [comment]
Thanks!

**Pull request merged**